src/test/java/FireDepartment.java

```diff
@@ -0,0 +1,32 @@
+public class FireDepartment {
+
+    private int countOfCall = 0;
+
+    public synchronized void extinguishTheFire() {
+
+        while (countOfCall < 1) {
+            try {
+                wait();
+            } catch (InterruptedException e) {
+            }
+        }
+        countOfCall--;
+        System.out.println("The fireman put out the fire");
+        System.out.println("The active fire is: " + countOfCall);
+        notify();
+    }
+
+    public synchronized void sendACall() {
+
```

Comment:
There is something wrong that call goes from the fire department. Don't you think so?

Reply:
Fixed